deps/rabbit/test/mc_SUITE.erl

```diff
@@ -0,0 +1,639 @@
+-module(mc_SUITE).
+
+-compile([export_all, nowarn_export_all]).
+
+-include_lib("rabbit_common/include/rabbit_framing.hrl").
+-include_lib("rabbit_common/include/rabbit.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("amqp10_common/include/amqp10_framing.hrl").
+-include_lib("rabbit/include/mc.hrl").
+
+%%%===================================================================
+%%% Common Test callbacks
+%%%===================================================================
+
+all() ->
+    [
+     {group, tests}
+    ].
+
+
+all_tests() ->
+    [
+     amqpl_defaults,
+     amqpl_compat,
+     amqpl_table_x_header,
+     amqpl_death_records,
+     amqpl_amqp_bin_amqpl,
+     amqp_amqpl,
+     stuff
+    ].
+
+groups() ->
+    [
+     {tests, [], all_tests()}
+    ].
+
+init_per_suite(Config) ->
+    Config.
+
+end_per_suite(_Config) ->
+    ok.
+
+init_per_group(_Group, Config) ->
+    Config.
+
+end_per_group(_Group, _Config) ->
+    ok.
+
+init_per_testcase(_TestCase, Config) ->
+    Config.
+
+end_per_testcase(_TestCase, _Config) ->
+    ok.
+
+%%%===================================================================
+%%% Test cases
+%%%===================================================================
+
+stuff(_Config) ->
+    MA = #'v1_0.message_annotations'{content = [{{symbol, <<"k">>}, {utf8, <<"v">>}}]},
+    % Desc = {described,
+    %         {utf8, <<"URL">>},
+    %         {utf8, <<"https://rabbitmq.com">>}},
+    MAEnc = amqp10_framing:encode(MA),
+    ct:pal("~p", [MAEnc]),
+    ct:pal("~p", [amqp10_framing:decode(MAEnc)]),
+
+    % amqp10_framing:decode(Desc),
+    ok.
```

Comment:
Should this be switched to an `?assertEqual(MA, amqp110_framing:decode(MAEnc))`? I assume this is for checking that encoding/decoding leads to the same terms